firefox-ios/Client/Frontend/Home/Homepage Rebuild/TopSites/TopSitesAction.swift

```diff
@@ -7,19 +7,27 @@ import Redux
 
 final class TopSitesAction: Action {
     var topSites: [TopSiteState]?
+    var numberOfRows: Int32?
```

Comment:
Why are we specifying an Integer size here?

Reply:
This is because I am dispatching an action that updates this property in `TopSitesSettingsViewController`, where I pass a value from the user defaults that returns an `Int32` type.

```
let numberOfRows = profile?.prefs.intForKey(PrefsKeys.NumberOfTopSiteRows) ?? defaultValue
```

Reply:
Yes, updated! commit here: https://github.com/mozilla-mobile/firefox-ios/pull/23918/commits/64b7a94a741269d361e71a1250b05ae777cbbf54